AdaptivePELE/freeEnergies/extractCoords.py

```diff
@@ -353,27 +370,30 @@ def main(folder_name=".", atom_Ids="", lig_resname="", numtotalSteps=0, enforceS
 
     # change atomId for list
 
-    if enforceSequential_run:
+    allFolders = os.listdir(folderWithTrajs)
+    folders = [epoch for epoch in allFolders if epoch.isdigit()]
+    if len(folders) == 0:
         folders = ["."]
-    else:
-        allFolders = os.listdir(folderWithTrajs)
-        folders = [epoch for epoch in allFolders if epoch.isdigit()]
-        if len(folders) == 0:
-            folders = ["."]
 
     if nProcessors is None:
         nProcessors = getCpuCount()
     nProcessors = max(1, nProcessors)
-    print("Running extractCoords with %d cores" % (nProcessors))
-    pool = mp.Pool()
 
+    if sidechains:
+        sidechains = extractSidechainIndexes(sidechain_folder, lig_resname)
+
+    print("Running extractCoords with %d cores" % (nProcessors))
+    if enforceSequential_run==1:
```

Comment:
Keep this block but with a new variable, for example parallelize or something like that, and keep the enforceSequential_run option for standard PELE compatibility. Also move lines 378-380 inside this block for clarity